operator/pkg/certs/certs.go

```diff
@@ -428,7 +429,8 @@ func ParsePrivateKeyPEM(keyData []byte) (crypto.Signer, error) {
 	case *ecdsa.PrivateKey:
 		key = k
 	default:
-		return nil, errors.New("the private key is neither in RSA nor ECDSA format")
+		keyType := fmt.Sprintf("%T", caPrivateKey)
+		return nil, fmt.Errorf("the private key is in an unsupported format: %s, supported formats are RSA and ECDSA", keyType)
```

Comment:
Could these two lines merged? like:

```suggestion
		return nil, fmt.Errorf("the private key is in an unsupported format: %T, supported formats are RSA and ECDSA", caPrivateKey)
```